docs/source/ecf.rst

```diff
@@ -1487,6 +1489,14 @@ pc_stepsize
 '''''''''''
 Computing uncertainties on the phase curve amplitude and offset can be slow; however, thinning the number of MCMC samples will speed up the calculation.  Increasing ``pc_stepsize`` to larger integer values will steadily decrease the computation time at the cost of accuracy.  Defaults to 50.  Use 1 for no thinning.
 
+strings_stepsize
+'''''''''''
+Same as ``pc_stepsize`` but applied to the Harmonica strings morning/evening limb calculation.  Defaults to 50.  Use 1 for no thinning.
+
+strings_angle
+'''''
+Harmonica strings angle (in degrees) to include in morning/evening limb calculation. The default angle of 60 degrees will span -30 to +30 degrees for the morning limb and 150 to 210 degrees for the evening limb.
```

Comment:
Might need an image or a link to the harmonica documentation, since this is a bit hard to understand.

Reply:
The Harmonica documentation is lacking in this respect, but Hannah says they are working on coming up a formalism for defining the optimal angle for a given set of planet parameters.  

In the mean time, I'll add a Stage 6 figure that we can also use here.

Reply:
Is this description more or less clear than before?
`The default angle of 60 degrees will span 0+/-30 degrees for the morning limb and 180+/-30 degrees for the evening limb.`

Reply:
Not really - having only briefly glanced at Harmonica a long time ago, I don't really know what all of this pertains to

Reply:
![HarmonicaStringAngle_60](https://github.com/user-attachments/assets/6eb28cef-7c16-4661-a560-0adabea9ecbd)
Thoughts?

Reply:
![HarmonicaStringAngle_60](https://github.com/user-attachments/assets/76a60d9a-7a57-41b2-bb29-0ea216e8a3ba)
Thoughts?

Reply:
Ahh okay, that makes things way easier to understand! Thanks!

What's the meaning behind the x-offset of the full transmission string and the reference circle?

Reply:
It's more than just an offset, it's the limb asymmetry.